checker/src/org/checkerframework/checker/formatter/FormatterAnnotatedTypeFactory.java

```diff
@@ -55,6 +58,12 @@ public FormatterAnnotatedTypeFactory(BaseTypeChecker checker) {
     }
 
     @Override
+    protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
+        return getBundledTypeQualifiersWithoutPolyAll(
+                UnknownFormat.class, FormatBottom.class);
```

Comment:
(Once #515 is complete, the default behavior will load UnknownFormat and FormatBottom)